### ISSUE TITLE
Just a reminder waiting for Espressif updates

### ISSUE THREAD

**Issue by PieBru** [esphome/feature-requests#390]
**Describe the problem you have/What new integration you would like**

ESP Wifi vulnerability: ESP8266 And ESP32 WiFi Hacked!
https://hackaday.com/2019/09/05/esp8266-and-esp32-wifi-hacked/

**Please describe your use case for this integration and alternatives you've tried:**

**Additional context**

**Comment by brandond** [comment]
Updates always seem to take a while, given they have to roll up through Espressif -> Arduino -> PlatformIO before we can even try using them in ESPHome.

**Comment by brandond** [comment]
Closing in favor of https://github.com/esphome/issues/issues/652
